include/mbedtls/md2.h

```diff
@@ -87,6 +117,11 @@ void mbedtls_md2_starts( mbedtls_md2_context *ctx );
  * \param ctx      MD2 context
  * \param input    buffer holding the  data
  * \param ilen     length of the input data
+ *
+ * \warning        MD2 is considered a weak message digest and its use
+ *                 constitutes a security risk. It is recommended
+ *                 to use a strong message digest instead.
```

Comment:
I think this warning is missing on `mbedtls_md2_finish()`

Reply:
Please note that this is not the only file that is missing warnings in some functions...